src/App.js

```diff
@@ -1,11 +1,14 @@
 import './App.css';
+import Page from './components/page';
+import Routing from './components/router';
 
 const App = () => {
-  return (
-    <div className='app'>
-        <div className='header'>Sw<span>oo</span>sh</div>
-    </div>
-  );
+
+    return <Page>
+        <div className={'appContainer'}>
```

Comment:
no need for curly braces around the className

Reply:
Done